vcf-dumper/vcf-dumper-lib/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporterController.java

```diff
@@ -135,29 +210,33 @@ public void run() {
         }
 
         writer.close();
+        logger.debug("VCF export finished: {} variants exported", totalExportedVariants);
```

Comment:
I'm unsure if that means 1000 were successfully exported and 20 failed, or 980 were successful and 20 failed... Maybe something more explicit, even in multiple messages, could be more clear?

>VCF export summary
>Variants processed: 1000
>Variants successfully exported: 1000 (or 980, or whatever)
>Variants with errors: 20